src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -32,23 +32,25 @@
      */
     @GetMapping(value = "/{slug}")
     public ResponseEntity<Object> getPostBySlug(@PathVariable("slug") String slug) {
-        Optional<PostEntity> postEntity = postService.findBySlug(slug);
+        PostDto postDto = postService.getPost(slug);
 
-        if (!postEntity.isPresent()) {
+        if (Objects.isNull(postDto)) {
             throw new PostNotFoundException("Cannot find post");
         }
 
-        SuccessResponse resp = new SuccessResponse(postEntity);
+        SuccessResponse resp = new SuccessResponse(postDto);
```

Comment:
postEntity is different from PostDto, changing this will affect the payload of the api

Reply:
Added some attribute in PostDTO so now PostDTO as same as PostEntity